include/nigiri/loader/gtfs/route.h

```diff
@@ -21,6 +21,8 @@ struct route {
   std::string long_name_;
   std::string desc_;
   clasz clasz_;
+  uint32_t color_;
```

Comment:
I tried to use `std::uint32_t` everywhere (at least in new code like `nigiri`) instead of `uint32_t`. I also think it makes sense to introduce a strong type, so it cannot get confused with a regular integer (e.g. in parameter lists).

```cpp
using color_t = cista::strong<std::uint32_t, struct _color>;
```